scripts/upgrade

```diff
@@ -106,18 +33,11 @@ then
 	ynh_script_progression --message="Upgrading source files..."
 
 	# Download, check integrity, uncompress and patch the source from app.src
-	ynh_setup_source --dest_dir="$final_path"
+	ynh_setup_source --dest_dir="$install_dir"
```

Comment:
Maybe adding `--keep="settings_base.json"` would fix the "Config file overwritten / manually modified" complain from the CI

```suggestion
	ynh_setup_source --dest_dir="$install_dir" --keep="settings_base.json"
```